resources/lang/de/validation.php

```diff
@@ -36,7 +36,7 @@
     'digits_between'       => ':attribute muss zwischen :min und :max Ziffern liegen.',
     'distinct'             => 'Das :attribute Feld hat einen doppelten Wert.',
     'email'                => ':attribute muss eine gültige E-Mail-Adresse sein.',
-    'exists'               => 'The selected :attribute is invalid.',
+    'exists'               => ':attribute ist ungültig.',
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal ":attribute ist ungültig." 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)